src/App.js

```diff
@@ -0,0 +1,39 @@
+import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { postRegister } from './redux/sessions/sessionsSlice';
+import Register from './components/sessions/register';
+import Login from './components/sessions/login';
+import Motorcycles from './components/pages/motorcycles';
+import Reservations from './components/pages/reservations';
+import AddMotorcycle from './components/pages/add-motorcycle';
+import Delete from './components/pages/delete';
+import Reserve from './components/pages/reserve';
+import Details from './components/pages/details';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found only 4):

![image](https://github.com/Bennyjoez/motorcycle-appointment-app-front-end/assets/57421823/83f697c1-a39b-44db-95ee-4d3d5c49cec4)


That way, your code will meet the latest standards. 💪